eml/api/api.js

```diff
@@ -42,15 +40,21 @@ export const getCourseWithAuth = async (courseId) => {
 
 export const getCourses = async () => {
   // TODO: add bearer token to request header and omit /public
-  const res = await axios.get(url + '/api/public/courses');
+  const res = await axios.get(url + '/api/courses');
+  return res.data;
+};
+
+export const getHome = async () => {
+  const res = await axios.get(url + '/api/course/getHome');
```

Comment:
Microsoft convention for endpoint says that http request types are not used in endpoints. 
So /api/course/home

Reply:
+1